modules/gdscript/gdscript_editor.cpp

```diff
@@ -56,15 +56,15 @@ Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const Str
 
 	String _template = String() +
 					   "extends %BASE%\n\n" +
-					   "# class member variables go here, for example:\n" +
+					   "# Class member variables go here, for example:\n" +
 					   "# var a = 2\n" +
 					   "# var b = \"textvar\"\n\n" +
 					   "func _ready():\n" +
 					   "%TS%# Called when the node is added to the scene for the first time.\n" +
 					   "%TS%# Initialization here.\n" +
```

Comment:
I think these two lines are redundant, plus `_ready()` isn't necessarily used for initialization, and "Initialization here" is an incomplete sentence.